### ISSUE TITLE
Error while running makemigrations

### ISSUE THREAD

**Issue by isamueljoshua** [ashishnitinpatil/django-oscar-cash-on-delivery#3]
Hi I am using 
Django 2.1.3 and Django-Oscar 1.6.4.After installing this app I tried to run the command `python manage.py makemigrations`. But, I got an error,
ModuleNotFoundError: No module named 'django.core.urlresolvers'

I guess this project does not support Django 2.0 and above?

**Comment by ashishnitinpatil** [comment]
Yes, the project currently does not support Django 2.0, and it lacks a test suite to test compatibility with Django, Django-oscar & Python versions properly. I'd suggest copying the app folder into your project and making required changes for now.

**Comment by Dihfahsih1** [comment]
Please consider updating the package so that it is easier for the open source community of coders, thanks for the effort of making django echo system so rich with libraries

**Comment by ashishnitinpatil** [comment]
Hi @Dihfahsih1 I'd consider the upgrade but cannot make any promises on the timeline. You're always free to submit a PR which I will review faster.
